framework/src/variables/MooseVariableFV.C

```diff
@@ -615,7 +626,7 @@ MooseVariableFV<OutputType>::getInternalFaceValue(const Elem * const neighbor,
   mooseAssert(isInternalFace(fi), "This function only be called on internal faces.");
 
   ADReal * value_pointer = &_temp_face_values[_tid];
-  if (_cache_face_values)
+  if (_cache_face_values & !correct_skewness)
```

Comment:
let s error out if cache_face_values is off when asking for correct_skewness.

We cant cache when the user requested no caching

Reply:
or is this the same scenario as the last one?

Reply:
Yes, we might want to cache uncorrected face values. It speeds up the skewness-correction for sure, since it computes: corr_face_val = uncorr_face_val + uncorr_face_grad_grad * diff_vector. In this case we save a little time if we cache the uncorrected face value.